File: lingting-datascope-jsql/src/main/java/live/lingting/framework/datascope/exception/DataScopeException.java
package live.lingting.framework.datascope.exception;

/**
 * @author lingting 2024-01-19 15:54
 */
public class DataScopeException extends RuntimeException {

	public DataScopeException(String message) {
		super(message);
	}

	public DataScopeException(String message, Throwable cause) {
		super(message, cause);
	}

}

File: lingting-datascope-jsql/src/main/kotlin/live/lingting/framework/datascope/exception/DataScopeException.kt
package live.lingting.framework.datascope.exception

/**
 * @author lingting 2024-01-19 15:54
 */
class DataScopeException : RuntimeException {
    constructor(message: String?) : super(message)

    constructor(message: String?, cause: Throwable?) : super(message, cause)
}
